main/acle.md

```diff
@@ -939,8 +939,8 @@ header:
 `<arm_fp16.h>` is provided to define the scalar 16-bit floating point
 arithmetic intrinsics. As these intrinsics are in the user namespace,
 an implementation would not normally define them until the header is
-included. The `__ARM_FEATURE_FP16_SCALAR_ARITHMETIC` feature macro
-should be tested before including the header:
+included. When `__ARM_FEATURE_FP16_SCALAR_ARITHMETIC` is defined to `1`,
+the header file is available in a normal unannotated function.
```

Comment:
The macro is also available outside of function scope. The scope in which it is used shouldn't matter, because it is a preprocessor macro.

Reply:
I wrote again this sentence to:
When `__ARM_xxx` is defined to `1`, the header file is available
regardless of the context in which the macro is evaluated.